### PR TITLE
Resolve some JBUGs in ast.bal

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
Related to #550

### CODE REVIEW

File: compiler/modules/front.syntax/ast.bal [hunk 1]
```diff
@@ -155,7 +155,9 @@ type CompoundAssignOp  "+=" | "-=" | "/=" | "*=" | "&=" | "|=" | "^=" | "<<=" |
 
 public type BinaryExprOp BinaryArithmeticOp|BinaryRelationalOp|BinaryEqualityOp;
 
-public type UnaryExprOp "-" | "!" | "~";
+public type NegateOp "-";
```
Comment (jclark) [inline compiler/modules/front.syntax/ast.bal]: Does

```
const NegateOp = "-";
```

work?

File: compiler/modules/front.syntax/ast.bal [hunk 2]
```diff
@@ -200,8 +202,7 @@ public type UnaryExpr record {|
 
 public type SimpleConstNegateExpr record {|
     *UnaryExpr;
-    // JBUG #32099 should be "-" 
-    UnaryExprOp op = "-";
+    NegateOp op = "-";
```
Comment (manuranga) [inline compiler/modules/front.syntax/ast.bal:206]: If you are unable to do `"-" op = "-";`, we still need a JBUG comment.

File: compiler/modules/front.syntax/ast.bal (hunk 1 above)
Comment (KavinduZoysa) [reply compiler/modules/front.syntax/ast.bal]: Yes, changed

**Comment by KavinduZoysa** [comment]
New issue:
https://github.com/ballerina-platform/ballerina-lang/issues/33369

**Review by jclark — APPROVED** [review]

**Pull request merged**